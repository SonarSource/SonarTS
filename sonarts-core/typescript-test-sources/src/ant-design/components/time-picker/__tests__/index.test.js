import React from 'react';
import { mount, render } from 'enzyme';
import RcTimePicker from 'rc-time-picker/lib/TimePicker';
import TimePicker from '..';

describe('TimePicker', () => {
  it('renders addon correctly', () => {
    const addon = () => (<button>Ok</button>);
    const wrapper = mount(<TimePicker addon={addon} />);
    const rcTimePicker = wrapper.find(RcTimePicker);
    const addonWrapper = render(rcTimePicker.props().addon());

    expect(addonWrapper).toMatchSnapshot();
  });
});
