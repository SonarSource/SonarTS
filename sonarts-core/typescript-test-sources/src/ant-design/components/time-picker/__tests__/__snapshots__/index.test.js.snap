// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimePicker renders addon correctly 1`] = `
<div
  class="ant-time-picker-panel-addon"
>
  <button>
    Ok
  </button>
</div>
`;
